test/unit/utils-deserialise-test.js

```diff
@@ -54,3 +58,21 @@ test('accounts with profile response', function (t) {
 
   t.end()
 })
+
+test('accounts with complex relationships', function (t) {
+  var data = deserialise(complexResponse, {
+    include: 'pet.vet'
+  })
+
+  t.deepEqual(data, complexReturn, 'returns right data')
+
+  t.end()
+})
+
+test('accounts with missing includes', function (t) {
+  var data = deserialise(accounts, options)
+
+  t.deepEqual(data, accountsReturn, 'returns right data')
+
+  t.end()
+})
```

Comment:
Hey @gr2m, do you mind having a look at this? I wrote these tests to achieve full branch coverage, but I have a hard time figuring out if the fixtures I'm using make sense and/or if this last test — with what to me looks like a malformed response — shouldn't throw errors.